pkg/metaclient/example

```diff
@@ -0,0 +1 @@
+kvclient/example.go
```

Comment:
What's this file used for?

Reply:
using mock kv client to show the usage of client interface